### PR TITLE
Add GDACS feed integration

### PR DESCRIPTION
**Pull request by exxamalte**
## Proposed change

Documents a new integration with the Global Disaster Alert and Coordination System (GDACS).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/31235
- This PR fixes or closes issue: n/a

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/gdacs.markdown [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+by distance to Home Assistant's location.
+
+Entities are generated, updated and removed automatically with each update 
+from the feed. Each entity defines latitude and longitude and will be shown 
+on the default map automatically, or on a map card by defining the source 
+`gdacs`. The distance is available as the state of each entity, and 
+converted to the unit (kilometers or miles) configured in Home Assistant.
+
+<p class='img'>
+  <img src='{{site_root}}/images/screenshots/gdacs-alerts-feed-map.png' />
```
Comment (frenck) [inline source/_integrations/gdacs.markdown]: Please don't use the `{{site_root}}`.

```suggestion
  <img src='/images/screenshots/gdacs-alerts-feed-map.png' />
```

File: source/_integrations/gdacs.markdown [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+## Sensor
+
+This integration automatically creates a sensor that shows how many entities
+are currently managed by this integration. In addition to that the sensor has
+some useful attributes that indicate the currentness of the data retrieved
+from the feed.
+
+<p class='img'>
+  <img src='{{site_root}}/images/screenshots/gdacs-alerts-sensor.png' />
```
Comment (frenck) [inline source/_integrations/gdacs.markdown]:
```suggestion
  <img src='/images/screenshots/gdacs-alerts-sensor.png' />
```

File: source/_integrations/gdacs.markdown [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+# Example configuration.yaml entry
+gdacs:
+```
+
+{% configuration %}
+categories:
+  description: The categories of alerts to be included. Valid categories are 'Drought', 'Earthquake', 'Flood', 'Tropical Cyclone', 'Tsunami', 'Volcano'.
+  required: false
+  type: list
+  default: None. Any alert regardless of its category will be included.
```
Comment (frenck) [inline source/_integrations/gdacs.markdown]: If there is no default, please remove it.
```suggestion
  default: None. Any alert regardless of its category will be included.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Almost perfect @exxamalte! 👍 

Could you take a look at the comments I left? Thanks!

File: source/_integrations/gdacs.markdown (hunk 1 above)
Comment (exxamalte) [reply source/_integrations/gdacs.markdown]: OK

File: source/_integrations/gdacs.markdown (hunk 2 above)
Comment (exxamalte) [reply source/_integrations/gdacs.markdown]: OK

File: source/_integrations/gdacs.markdown (hunk 3 above)
Comment (exxamalte) [reply source/_integrations/gdacs.markdown]: OK

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — APPROVED** [review]

**Pull request merged**